### PR TITLE
Keep track of the processing time for XMPP requests.

### PR DESCRIPTION
**Pull request by bgrozev**
Depends on jitsi/jitsi-utils#45 and jitsi/jitsi-media-transform#217

### CODE REVIEW

File: src/main/java/org/jitsi/videobridge/xmpp/XmppCommon.java [hunk 1]
```diff
@@ -214,6 +239,16 @@ else if (request instanceof ShutdownIQ)
...
             response = null;
         }
+
+        if (delayStats != null)
+        {
+            long delay = System.currentTimeMillis() - start;
+            if (delay > 100)
+            {
+                logger.warn("Took " + delay + "ms to handle IQ: " + request.toXML());
+            }
```
Comment (JonathanLennox) [inline src/main/java/org/jitsi/videobridge/xmpp/XmppCommon.java]: Missing space before `"ms`?

**Comment by bbaldino** [comment]
Should we expose these through datadog as well?

**Comment by bbaldino** [comment]
(Sorry, closed on accident)

File: src/main/java/org/jitsi/videobridge/xmpp/XmppCommon.java (hunk 1 above)
Comment (bgrozev) [reply src/main/java/org/jitsi/videobridge/xmpp/XmppCommon.java]: No, that was intentional. I've *always* formatted it this way! Looks like I've always been wrong :)

**Comment by bgrozev** [comment]
> Should we expose these through datadog as well?

I think this wouldd be part of the infra scripts. I'll ping you about it

**Pull request merged**